include/deal.II/numerics/vector_tools.h

```diff
@@ -1858,41 +1900,41 @@ namespace VectorTools
    *
    * See the general documentation of this namespace for further information.
    */
-  template <int dim, int spacedim>
+  template <int dim, typename VectorType, int spacedim>
   void create_right_hand_side (const Mapping<dim, spacedim>    &mapping,
                                const DoFHandler<dim,spacedim>  &dof,
                                const Quadrature<dim>           &q,
                                const Function<spacedim,double> &rhs,
-                               Vector<double>                  &rhs_vector);
+                               VectorType                      &rhs_vector);
```

Comment:
Why do you insert the `VectorType` between the `dim` and `spacedim` template arguments? We typically put the integer templates in front.